vignettes/age-struct-pop.Rmd

```diff
@@ -0,0 +1,206 @@
+---
+title: "Age structured population"
+output: 
+  bookdown::html_vignette2:
+    code_folding: show
+pkgdown:
+  as_is: true
+vignette: >
+  %\VignetteIndexEntry{Age structured population}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+::: {.alert .alert-info}
+If you are unfamiliar with the {simulist} package or the `sim_linelist()` function [Get Started vignette](simulist.html) is a great place to start.
+:::
+
+This vignette describes how to simulate a line list with either a uniform or non-uniform distribution of ages within the population. This population age structure may either describe the population of a region as a whole, or specifically which age groups are more likely to become infected than others, for instance because they have more social contacts.
+
+The {simulist} package uses a branching process which is independent of population age structure to simulate the line list, and then {simulist} _paints_ the demographic information onto the infected individuals (and in the case of `sim_outbreak()` or `sim_contacts()` the contacts of infected individuals).
+
+The default age structure in {simulist} is a uniform structure between a lower and upper age range. The `sim_linelist()` function (and other `sim_*()` functions) can accept a `<data.frame>` instead of the `numeric` vector to specify the age structure for specified age groups. This feature can be especially useful when wanting to simulate an outbreak in a region with a heavily non-uniform age structure, [for example younger populations such as Nigeria, or older populations such as Japan](https://ourworldindata.org/age-structure). 
+
+```{r setup}
+library(simulist)
+library(epiparameter)
+library(ggplot2)
+```
+
+First we load the requested delay distributions using the {epiparameter} package.
+
+```{r, read-delay-dists}
+serial_interval <- epidist(
```

Comment:
here again it'd be worth mentioning that this is defined rather than extracted because it's not in the database, in case this is the first vignette that someone reads

Reply:
Resolved in 2304a35f9c1a1fe890c16b0eb7a5ba6ed0cedecf.